pages/1.13/monitoring/logging/aggregating/elk/index.md

```diff
@@ -10,144 +10,82 @@ enterprise: false
 
 
 
-You can pipe system and application logs from the nodes in a DC/OS cluster to an Elasticsearch server. These instructions are based on CentOS 7 and might differ substantially from other Linux distributions.
+You can pipe system and application logs from the nodes in a DC/OS cluster to an Elasticsearch server.
 
 
 # What this document does and does not cover
 
-This document describes how to send Filebeat output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Filebeat into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][8] and the example in [Filtering logs with ELK][3].
+This document describes how to send Fluent Bit output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Fluent Bit into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][4] and the example in [Filtering logs with ELK][2].
 
-This document does not explain how to set up and configure an Elasticsearch server. This document does not describe how to set up secure TLS communication between the Filebeat instances and Elasticsearch. For details on how to achieve this, see the [Filebeat][2] and [Elasticsearch][5] documentation.
+This document does not explain how to set up and configure an Elasticsearch server. This document does not describe how to set up secure TLS communication between the Fluent Bit instances and Elasticsearch. For details on how to achieve this, see the [Fluent Bit][1] and [Elasticsearch][3] documentation.
```

Comment:
<p class="message--warning"><strong>WARNING: </strong>Instructions on how to set up and configure an Elasticsearch server, secure TLS communication between the Fluent Bit instances and Elasticsearch are not explained here. You can see the [Fluent Bit][1] and [Elasticsearch][3] documentation for more information to achieve this task. </p>